scrapers/vt/bills.py

```diff
@@ -9,6 +9,40 @@
 
 
 class VTBillScraper(Scraper, LXMLMixin):
+
+    # Dictionary matching bill action phrases to classifications
+    _actions = {
+        "signed by governor": ["executive-signature"],
+        "become law without signature of governor": ["became-law", "executive"],
+        "vetoed by the governor": ["executive-veto"],
+        "read first": ["introduction"],
+        "read 1st time": ["introduction"],
+        "first read": ["introduction"],
+        "reported favorably": ["committee-passage-favorable"],
+        "read second": ["reading-2"],
+        "read 2nd": ["reading-2"],
+        "second read": ["reading-2"],
+        "read third": ["reading-3"],
+        "read 3rd": ["reading-3"],
+        "third read": ["reading-3"],
+        "rules suspended & messaged to house forthwith": ["passage"],
+        "overrid": ["veto-override-passage"],
+        "delivered to governor": ["executive-receipt"],
+        "vetoed": ["executive-veto"],
+        "governor allowed to become law": ["became-law"],
+        "allowed to go into effect without the signature of the governor": [
+            "became-law"
+        ],
+        "proposal of amendment": ["amendment-introduction"],
+    }
+
+    def categorize_actions(self, action_string):
+        for action_key in list(self._actions):
```

Comment:
`for action_key in self._actions.keys()`?

Reply:
good point! will change